admin/environment-management/cvms.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Docker in Environments
+description: Learn how to enable support for secure Docker inside Environments. 
+---
+
+As a platform administrator, you have the option to enable
+[Container-based Virtual Machines](../../environments/cvms.md) as an Environment
+deployment option. This option allows users to run system-level
+programs inside their Environment including Docker and systemd.
+
+Navigate to **Manage > Admin > General** to enable this option.
+
+## Infrastructure requirements
+
+- The Kubernetes Nodes must have a minimum kernel version of `5.4`
+(released Nov 24th, 2019).
+- The Kubernetes Nodes must be running an Ubuntu OS.
```

Comment:
Is this a sysbox restriction or are we trying to limit the OSes we need to support?

Reply:
@kylecarbs doesn't think this is *needed* but if I recall correctly the sysbox maintainers suggested it as a requirement. So including it in our docs feels like the safe bet either way.

Reply:
So with the `5.4` restriction I don't believe we need to limit to Ubuntu distributions... I can say though that this is all we've tested on, so might be risky to do anything else.

Reply:
agree, let's specify it and someone can ask for another distro if they want it – at that point we can test it out if it makes sense